main_logic/core.py

```
@@ -569,6 +580,38 @@ async def handle_response_discarded(self, reason: str, attempt: int, max_attempt
             except Exception as e:
                 logger.warning(f"发送 response_discarded 到前端失败: {e}")
 
+        # RESPONSE_TOO_LONG 最终丢弃时：发送可爱回复 + 用角色 TTS 音色念出来
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)